lib/kb_Msuite/Utils/DataStagingUtils.py

```diff
@@ -81,6 +106,21 @@ def stage_input(self, input_ref, fasta_file_extension):
         return {'input_dir': input_dir, 'folder_suffix': suffix, 'all_seq_fasta': all_seq_fasta}
 
 
+    def count_fasta_seq_len(self, fasta_path):
+        '''
+        counts the number of non-header, non-whitespace characters in a FASTA file
+        '''
+        seq_len = 0
```

Comment:
I know it's not hard to come up your own logic but you should consider using tested package such as Bio.SeqIO.

Reply:
I've been writing sequence parsers since 1999.  I don't want to pay the cost of importing a library for something trivial.  Also, SeqIO is kind of inefficient.